Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/activities/EmulationActivity.java

```diff
@@ -486,13 +513,40 @@ public boolean onKeyLongPress(int keyCode, @NonNull KeyEvent event)
   protected void onActivityResult(int requestCode, int resultCode, Intent result)
   {
     super.onActivityResult(requestCode, resultCode, result);
-    if (requestCode == REQUEST_CHANGE_DISC)
+    if (resultCode == RESULT_OK)
     {
-      // If the user picked a file, as opposed to just backing out.
-      if (resultCode == RESULT_OK)
+      if (requestCode == REQUEST_CHANGE_DISC)
       {
+        // If the user picked a file, as opposed to just backing out.
```

Comment:
The new placement of this comment doesn't make sense. It's referring to the `RESULT_OK` check.